.github/workflows/trigger_windows.yml

```diff
@@ -0,0 +1,75 @@
+name: Build Windows Images
+
+on:
+  push:
+    paths: [.github/**, images/**]
+    branches: [ "main" ]
+  pull_request:
+    paths: [.github/**, images/**]
+    branches: [ "main" ]
+
+  workflow_dispatch:
+
+jobs:
+  build-all-images:
+    name: Build Windows images
+    strategy:
+      fail-fast: false
+      matrix:
+        runner: [windows-2019, windows-2022]
+        cuda_version: ["12.1", "11.1"]
+        edition: [windows-2019, windows-2022]
```

Comment:
Runner/edition can be different in the future. We'll likely refactor this before a new edition of GHA runners comes out.